tests/TestHelpers/GraphHelper.php

```diff
@@ -1543,11 +1543,12 @@ public static function getPermissionsList(
 	/**
 	 * @param string $baseUrl
 	 * @param string $xRequestId
-	 * @param string $user
-	 * @param string $userIdOfShareeUser
+	 * @param string $sharer
+	 * @param string $userIdOfShareeUser this is a uuidv4 of sharer
```

Comment:
then, also need to update the parameter name as well from sharee to sharer

Reply:
that's actually of sharee user